server/src/lib/fetch-lighthouse.ts

```diff
@@ -21,7 +21,6 @@ export async function fetchLighthouse(url: string): Promise<LighthouseResults> {
   }
 }
 
```

Comment:
Can we get this out of this PR, we should be only working on migration validator for now